TheMovieDb/Models/MoviesList.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MovieList.swift
+//  TheMovieDb
+//
+//  Created by Alex on 11/4/21.
+//
+
+import Foundation
+
+struct MovieItem: Decodable {
+    let id: String
+    let name: String
+
+    enum CodingKeys: String, CodingKey {
+        case id
+        case name
+        case url
+    }
+    
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
```

Comment:
I believe that we can simplify the decoding process a bit. We'll talk about it in the mentoring session. 🙂